src/modules/job-manager/event.c

```diff
@@ -554,6 +581,48 @@ static int get_timestamp_now (double *timestamp)
     return 0;
 }
 
+
+/*  Call jobtap plugin for event if necessary.
+ *  Currently jobtap plugins are called only on state transitions or
+ *   an asynchronous urgency update.
```

Comment:
I meant an update to urgency can occur through the RPC, yes, though this could occur outside of a user requested update. I'll try to make this more clear.